src/components/VideoProvider/useHandleRoomDisconnectionErrors/useHandleRoomDisconnectionErrors.ts

```diff
@@ -2,12 +2,13 @@ import { Room, TwilioError } from 'twilio-video';
 import { useEffect } from 'react';
 import { ROOM_STATE } from '../../../utils/displayStrings';
 import { Callback } from '../../../types';
+import { NOTIFICATION_MESSAGE } from '../../../utils/displayStrings';
 
-export default function useHandleRoomDisconnectionErrors(room: Room, onError: Callback) {
+export default function useHandleRoomDisconnectionErrors(room: Room, onError: Callback, onNotification: Callback) {
```

Comment:
useHandleRoomDisconnectionErrors I think would benefit from being renamed to useHandleRoomDisconnectionEvents to allow its meaning to be handling of both errors and notifications